eth2util/eth2exp/attagg.go

```diff
@@ -17,17 +17,26 @@ package eth2exp
 
 import (
 	"context"
+	"encoding/binary"
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
 	"strconv"
 	"strings"
 
+	eth2client "github.com/attestantio/go-eth2-client"
 	eth2p0 "github.com/attestantio/go-eth2-client/spec/phase0"
 
 	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/eth2util"
 )
 
+type eth2Provider interface {
```

Comment:
rather just export this and take it as the parameter for `Calculate`, we do not need to follow the cast pattern anymore since we now use `eth2wrap.Client` that implements everything.

Reply:
@corverroos why do we need to export this?